### PR TITLE
Add Quine in Rust

### LINKED ISSUE #537: Add Quine in Rust Article

### PR DESCRIPTION
**Pull request by rzuckerm**
Thanks for helping us build up our knowledge base! 
What kind of pull request have you brought us today?

## Pull Request Type

- [x] Sample Program Documentation
- [ ] Language Documentation
- [ ] Project Documentation
- [ ] Other (explain below)

## Notes

I fixed #537

### CODE REVIEW

File: sources/programs/quine/rust/how-to-run-the-solution.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+If you want to run this program, you'll first need to download a
+copy of the 
+[Rust compiler](https://www.rust-lang.org/tools/install),
+and follow the installation instructions. From there, open a terminal, and
+run this command:
+
+```
+rustc quine.eu
```
Comment (jrg94) [inline sources/programs/quine/rust/how-to-run-the-solution.md:8]: Is the extension meant to be .eu?

**Review by jrg94 — APPROVED** [review]
Really nice article! Excited to get this in front of more eyes.

File: sources/programs/quine/rust/how-to-run-the-solution.md (hunk 1 above)
Comment (rzuckerm) [reply sources/programs/quine/rust/how-to-run-the-solution.md:8]: Oops, you caught me! I copy this file from Euphoria. I'll fix that

Comment (rzuckerm) [inline sources/programs/quine/rust/how-to-run-the-solution.md:8]:
```suggestion
rustc quine.rs
```

**Pull request merged**